stdlib/list.ml

```diff
@@ -66,9 +66,15 @@ let rec init_aux i n f =
     let r = f i in
     r :: init_aux (i+1) n f
 
+let rev_init_threshold =
+  match Sys.backend_type with
+  | Sys.Native | Sys.Bytecode -> 10_0000
```

Comment:
I guess it was supposed to be `10_000`, not `10_0000`...

Reply:
Indeed, good catch... It is a bit disconcerting that our CI didn't catch that regression.

Reply:
I fixed this in fa884f420b257f350a5a4a3e701b5acd1df9c10a.